### PR TITLE
chore(docs): add documentation to GitHub Scraper defining PAT permissions

### PR DESCRIPTION
**Pull request by gnmeyer**

### CODE REVIEW

File: receiver/gitproviderreceiver/README.md [hunk 1]
```diff
@@ -87,6 +87,19 @@ service:
...
+* read:packages
+* read:org
+* read:user
+* read:enterprise
+* read:project
+* read:security
+
+The queries under the hood are all read queries, there is no write. They read repos, orgs, branches, commits, metadata, issues, pull requests, and GHAS security events.
+
+It is recommend to use the [GitHub App Authentication Extension](https://github.com/liatrio/liatrio-otel-collector/tree/main/extension/githubappauthextension) for authentication because GitHub Apps use OAUTH and are more secure, locks down permissions better, and increases rate limits.
```
Comment (jburns24) [inline receiver/gitproviderreceiver/README.md:101]:
```suggestion
It is recommend to use the [GitHub App Authentication Extension](https://github.com/liatrio/liatrio-otel-collector/tree/main/extension/githubappauthextension) for authentication. GitHub Apps offer better security through OAUTH, provide more granular control over permissions, and increase rate limits.
```

File: receiver/gitproviderreceiver/README.md [hunk 2]
```diff
@@ -87,6 +87,19 @@ service:
             exporters: [...]
 ```
 
+### Github PAT basic permissions
+* repo:All
```
Comment (adrielp) [inline receiver/gitproviderreceiver/README.md:91]: Shouldn't it just be `repo:read` or `read:repo`? There's nothing in here that writes to GitHub as all calls are read only.

**Review by adrielp — CHANGES_REQUESTED** [review]
A couple changes from myself and @jburns24. Once those are made then it'll be good to go!

File: receiver/gitproviderreceiver/docs/repo-perms-3.png
Comment (adrielp) [inline receiver/gitproviderreceiver/docs/repo-perms-3.png:1]: Do we not also need secret scanning alerts? 

Was this copy-pasta from the GitHub App Auth extension? If so, App auth is not 1 -> 1 with Fine grained tokens IIRC.

Comment (gnmeyer) [reply receiver/gitproviderreceiver/docs/repo-perms-3.png:1]: These are the differing permissions offered between App Auth and Fine Grained Tokens. Ive updated the screenshot to show the list of permissions from the Fine Grained Token page.

| App Auth   | Fine Grained |
| ---------- | ------------ |
| Checks     | NA           |
| NA         | Custom properties |
| Packages   | NA           |
| Project    | NA           |
| Single File| NA           | 

git.repository.cve.count
The number of Common Vulnerabilities and Exposures (CVEs) in the repository.

This metric is covered by the [Repository security advisories](https://docs.github.com/en/rest/authentication/permissions-required-for-fine-grained-personal-access-tokens?apiVersion=2022-11-28#repository-permissions-for-repository-security-advisories) and [Dependabot Alerts](https://docs.github.com/en/rest/authentication/permissions-required-for-fine-grained-personal-access-tokens?apiVersion=2022-11-28#repository-permissions-for-dependabot-alerts)

**Review by adrielp — APPROVED** [review]

**Pull request merged**